app/models/google_event.rb

```diff
@@ -137,8 +137,13 @@ def event_declined?(event)
     end
 
     def normalize_event_datetime(event)
-      event.start.date_time = EventGrouper.floor_time(event.start.date_time)
-      event.end.date_time = EventGrouper.ceil_time(event.end.date_time)
+      if event.start.date_time
```

Comment:
how about having a method called `full_day_event?`. It would clarify why you need to do such things